release notes/v0.40.0.md

```diff
@@ -0,0 +1,249 @@
+k6 v0.40.0 is here! :tada: 
+#TODO 
+[Roadmap and future plans section](#roadmap-and-future-plans).
+
+## New Features and notable changes!
+
+### Breaking change!: Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
+
+As part of the ESM changes it was found out that anything defined in the scope of the main module also was accessible globally. This was because it was directly evaluated in the global scope.
+
+This has now been remedied and is no longer the case. This is a *breaking* change, but given that the whole point of modules, CommonJS or ESM, is to separate modules - so this is obviously a bug.
+
+We already have reports by people who have knowingly or not used it, but it seems fairly rare and contained to one or two usages in a script. The recommended way is to if you need something globally to set it explicitly on the global object `globalThis`.
```

Comment:
```suggestion
As part of the ESM changes, it was found that anything defined in the main module's scope was also accessible globally. This was because it was directly evaluated in the global scope.
This has now been remedied and is no longer the case. This is a *breaking* change, but given that the whole point of modules, CommonJS or ESM, is to separate modules, this is obviously rather a bug than a feature.

On the other note, we've seen reports by people who have this global accessibility of the main module (intentionally or not). Still, it seems relatively rare, and only a few usages in a script. So if you need something accessed globally, our suggested workaround is to set it explicitly on the global object `globalThis`.
```